src/main/java/no/nav/familie/inntektsmelding/typer/SaksnummerDto.java

```diff
@@ -2,29 +2,15 @@
 
 import java.util.Objects;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Pattern;
 
-public final class SaksnummerDto {
+public record SaksnummerDto (@JsonValue @NotNull @Pattern(regexp = REGEXP, message = "Saksnummer [${validatedValue}] matcher ikke tillatt pattern [{regexp}]") String saksnr) {
 
     private static final String REGEXP = "^[\\p{Alnum}]+$";
 
-    @NotNull
-    @Pattern(regexp = REGEXP, message = "Saksnummer [${validatedValue}] matcher ikke tillatt pattern [{regexp}]")
-    private String saksnr;
-
-    public SaksnummerDto() {
-    }
-
-    public SaksnummerDto(String saksnr) {
-        Objects.requireNonNull(saksnr);
-        this.saksnr = saksnr;
-    }
-
-    public String getSaksnr() {
```

Comment:
Eg har tidligere hatt problemer med at Jackson ikkje klarer å deserialisere dersom det ikkje finnes ein get-metode

Reply:
Jackson har støtte for records - så det bør ikke være et problem lenger. Skal lage en serdes test.

Reply:
Da er testen på plass.